commons/commons-bio/src/main/kotlin/ebi/ac/uk/model/MigrateHomeOptions.kt

```diff
@@ -0,0 +1,6 @@
+package ebi.ac.uk.model
+
+data class MigrateHomeOptions(
+    val storageMode: String,
+    val onlyEmptyFolder: Boolean = true,
```

Comment:
I'd suggest `onlyIfEmptyFolder` since the current name suggests the task will only migrate the empty folders and not that it will fail based on whether the folder is empty

Reply:
totally agree